### ISSUE TITLE
Reconsider pilcrows

### ISSUE THREAD

**Issue by mnot** [rfc-format/draft-iab-html-rfc-bis#19]
I got some private feedback from someone that the approach that the document takes to pilcrows is _horrible_ for accessibility.

More generally, we should really get some accessibility folks to review what we produce...

**Comment by reschke** [comment]
> ... that the approach that the document takes to pilcrows is horrible for accessibility.

how so? Details appreciated.

**Comment by mnot** [comment]
Not sure; I encouraged them to create an issue. I suspect they're read out by screen readers, etc...

**Comment by reschke** [comment]
FWIW, I just tried with rfc9001.html on Edge's screen reader and it does not read them. More info needed.
